15/umbraco-deploy/installation/install-configure.md

```diff
@@ -24,9 +24,9 @@ Here we will cover how to install and set up Umbraco Deploy on a new website.
 
 **Set up the Git repository and Umbraco project**
 
-The first step to get Umbraco Deploy up and running is to set up a GitHub repository. This will be where source code is stored, and, following the GitHub actions example, act as our environment where we will set up a CI/CD pipeline.
+The first step to get Umbraco Deploy up and running is to set up a GitHub repository. This will be where source code is stored, and, following the GitHub Actions example, act as our environment where we will set up a CI/CD pipeline.
```

Comment:
```suggestion
The first step to get Umbraco Deploy up and running is to set up a GitHub repository. This will store the source code and serve as our environment for setting up a CI/CD pipeline, following the GitHub Actions example.
```